Templates/Ros2FleetRobotTemplate/README.md

```diff
@@ -0,0 +1,132 @@
+# ROS2 Fleet Robot Template
+
+A project template that allows building a multi-robot ROS2-enabled simulation. The template contains a level with a large warehouse scene called `Warehouse.prefab`. 
+
+## Requirements
+
+Due to ROS2 dependency, this project was prepared and tested on Linux operating system. It is recomended to use Ubuntu [22.04](https://www.releases.ubuntu.com/22.04/), however any distribution meeting following requirements may be used.
+
+Refer to the [O3DE System Requirements](https://www.o3de.org/docs/welcome-guide/requirements/) documentation to make sure that the system/hardware requirements are met. 
+This project has the following dependencies:
+
+- [O3DE](https://github.com/o3de/o3de)
+- [ROS2 Humble](https://docs.ros.org/en/humble/index.html)
+- [ROS2 Gem](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2)
+- [Proteus robot](https://github.com/o3de/o3de-extras/tree/development/Gems/ProteusRobot)
+- [WarehouseAssets](https://github.com/o3de/o3de-extras/tree/development/Gems/WarehouseAssets)
+
+Please make sure that `clang` was installed and configured. For details refer to [this section](https://www.o3de.org/docs/welcome-guide/requirements/#linux) of O3DE documentation.
+
+## Setup Instructions
+
+The following steps will assume the following:
+
+- You have ROS2 humble [installed](https://docs.ros.org/en/humble/Installation.html) and environment is [sourced](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#source-the-setup-files).
+- The O3DE gems, projects and templates will be placed in the O3DE home folder: `${HOME}/O3DE`.
+- You have the O3DE engine [downloaded and built](https://www.o3de.org/docs/welcome-guide/setup/setup-from-github) and it is located in `/home/${USER}/O3DE/Engines/Development`.
+
+In this tutorial, CLI tools will be used. It is also possible to use O3DE GUI to set up a project from template. See the [O3DE Project manager documentation](https://www.o3de.org/docs/user-guide/project-config/project-manager/) for more details. In such cases it is required to source your ROS2 distro before launching O3DE manager:
+
+```shell
+source /opt/ros/humble/setup.bash
+./build/linux/bin/profile/o3de
+```
+
+### 1. Install ROS2 packages
+
+```shell
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-gazebo-msgs ros-${ROS_DISTRO}-control-toolbox
```

Comment:
These should be installed through the Gem instructions, no additional packages are needed for the Template. Again, this could be covered by #111 though

Reply:
Since #111 is ongoing, I would merge it as is. 
Next, a PR for this reamde and another one (https://github.com/o3de/o3de-extras/blob/development/Templates/Ros2ProjectTemplate/README.md) should be created.